.github/workflows/maven.yml

```diff
@@ -16,20 +16,16 @@ jobs:
 
     steps:
     - name: Repository checkout
-      uses: actions/checkout@v2
-    - name: Cache local Maven repository
-      uses: actions/cache@v2
+      uses: actions/checkout@v3
+    - name: Set up JDK 8
+      uses: actions/setup-java@v3
       with:
-        path: ~/.m2/repository
-        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-        restore-keys: ${{ runner.os }}-maven-
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.8
+        java-version: 8
+        distribution: 'adopt'
+        cache: maven
```

Comment:
Testing if we can use the default 'cache' mechanism, might cause the dependency-check to not be cached though.

Reply:
took 3 minutes the first time

Reply:
Took 2m 10s second time. Does not seem like it is caching.

Reply:
Old caching method + new other actions seem to work fine, dependency-check caching is back as well.